includes/tracks/events/class-wc-admin-setup-wizard-tracking.php

```diff
@@ -49,4 +52,13 @@ public static function track_store_setup() {
 
 		WC_Tracks::record_event( 'obw_store_setup', $properties );
 	}
+
+	/**
+	 * Tracks when Jetpack is activated through the OBW.
+	 *
+	 * @return void
+	 */
+	public static function track_activate() {
```

Comment:
`track_jetpack_activate` ?

Reply:
Sorry, merged before I saw your comment here.  Will add this change to https://github.com/woocommerce/woocommerce/pull/22877